CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Expand audit logging.
 - Enable DynamicAuditing feature gate.
 
+### Changed
+
+- Update Kubernetes to `1.19.7` version.
+- Update Flatcar Linux to `2605.12.0` version.
+
+### Deleted
+
+- Remove `--audit-dynamic-configuration` flag from Kubernetes API server as it is not supported in `1.19.x`.
```

Comment:
That's funny. We also have `Enable DynamicAuditing feature gate` in the same changelog. Should we remove both entries instead since it wasn't released?

Reply:
cc @stone-z

Reply:
removed